src/api/java/com/minecolonies/api/configuration/ServerConfiguration.java

```diff
@@ -443,15 +442,11 @@ protected ServerConfiguration(final ForgeConfigSpec.Builder builder)
             "forge:string"),
           s -> s instanceof String);
 
-        guardResourceLocations = defineList(builder, "guardresourcelocations",
```

Comment:
want to remove the lang entry for `guardresourcelocations` aswell I guess

Reply:
done